src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -71,12 +73,15 @@ public long id() {
         throw new UnsupportedOperationException("id() not yet implemented");
     }
 
-    // @todo #15:30min Implement time() by parsing the string representation
-    //  of transaction according to the pattern, described in the white
-    //  paper. Replace relevant test case with actual tests.
     @Override
-    public ZonedDateTime time() {
-        throw new UnsupportedOperationException("time() not yet implemented");
+    public ZonedDateTime time() throws IOException {
+        return new ZonedDateTimeOf(
+            new IoCheckedScalar<>(
+                () -> new ItemAt<>(
```

Comment:
@carlosmiranda directly passe the `ItemAt` to the containing `Scalar` or via an `UncheckedText` to avoid using lambda.